raiden/tests/fixtures/matrix.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+
+from mirakuru import TCPExecutor
+
+
+@pytest.fixture
+def use_local_matrix_server():
+    """
+    If True, try to test with a real client and start a fresh local
+    matrix server for each test. This should be used for integration
+    tests only.
+    If False, use the mock matrix client.
+    """
+    return False
+
+
+# the empty fixtures are overwritten in integration/conftest.py
+@pytest.fixture
+def local_matrix():
```

Comment:
Why do you do that? Why have empty fixtures here and rewrite them in conftestpy?

Reply:
The values of these fixtures can be set with command line options. I could have given the default values here, rather than in conftest.py, but I thought it's better to have the option names and default values in one place.

Reply:
That's going against our style. In the rest of the codebase we give the default value in the fixture declaration itself. Will change it later, but not in this PR as I want it merged soon.